src/redux/actions/getDeployments.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { GET_DEPLOYMENTS } from './actionTypes';
+import { API_BASE_URL } from '../../config';
+
+const getDeployments = (clusterID) => (dispatch) => {
+  axios.get(`${API_BASE_URL}/clusters/${clusterID}/deployments`)
+    .then((response) => {
+      console.log(response.data.data.deployments);
+      debugger;
+      dispatch({
+        type: GET_DEPLOYMENTS,
+        payload: response.data.data.deployments
+      });
+    })
+    .catch((error) => {
+      console.log(error);
```

Comment:
We could have an action that handles the error returned instead of a console log given this doesnt cater for users incase an error is returned by the backend end and a user requires an appropriate response.